### PR TITLE
TiUP: Add three docs about tiup dm display, upgrade, and cluster check

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: pingcap/docs-cn#4868
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tiup/tiup-component-dm-upgrade.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+The `tiup dm upgrade` command is used to upgrade a specified cluster to a specific version.
+
+## Syntax 
+
+```shell
+tiup dm upgrade <cluster-name> <version> [flags]
+```
+
+- `<cluster-name>` is the name of the cluster to operate on. If you forget the cluster name, check it in [tiup cluster list](/tiup/tiup-component-cluster-list.md).
```
Comment (CharLotteiu) [inline tiup/tiup-component-dm-upgrade.md:15]:
```suggestion
- `<cluster-name>` is the name of the cluster to be operated on. If you forget the cluster name, check it in [tiup DM cluster list](/tiup/tiup-component-dm-list.md).
```

File: tiup/tiup-component-dm-upgrade.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+### -h, --help
+
+- Prints the help information.
+- Data type: `BOOLEAN`
+- This option is disabled by default with the `false` value. To enable this option, add this option to the command, and either pass the `true` value or do not pass any value.
+
+## Output
+
+Log of the upgraded service.
```
Comment (CharLotteiu) [inline tiup/tiup-component-dm-upgrade.md:28]:
```suggestion
Log of the service upgrade process.
```

File: tiup/tiup-component-dm-upgrade.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+The `tiup dm upgrade` command is used to upgrade a specified cluster to a specific version.
+
+## Syntax 
+
+```shell
+tiup dm upgrade <cluster-name> <version> [flags]
+```
+
+- `<cluster-name>` is the name of the cluster to operate on. If you forget the cluster name, check it in [tiup cluster list](/tiup/tiup-component-cluster-list.md).
+- `<version>` is the target version to upgrade to. Currently, only upgrading to a later version is allowed, and upgrading to an earlier version is not allowed, which means the downgrade is not allowed. Upgrading to a nightly version is not allowed either.
```
Comment (CharLotteiu) [inline tiup/tiup-component-dm-upgrade.md:16]:
```suggestion
- `<version>` is the target version to be upgraded to. Currently, only upgrading to a later version is allowed, and upgrading to an earlier version is not allowed, which means the downgrade is not allowed. Upgrading to a nightly version is not allowed either.
```

File: tiup/tiup-component-dm-display.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+- Data type: `BOOLEAN`
+- This option is disabled by default with the `false` value. To enable this option, add this option to the command, and either pass the `true` value or do not pass any value.
+
+## Output
+
+- Cluster name
+- Cluster version
+- SSH client type
+- A table containing the following fields:
+    - `ID`: the Node ID, consisting of IP:PORT.
```
Comment (CharLotteiu) [inline tiup/tiup-component-dm-display.md:51]:
```suggestion
    - `ID`: the node ID, consisting of IP:PORT.
```

File: tiup/tiup-component-dm-display.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+Specifies the IDs of the nodes to query, splitting by commas for multiple nodes. If you are not sure about the ID of a node, you can skip this option in the command to show the IDs and status of all nodes in the output.
+Data type: `STRING`
+This option is enabled by default with `[]` (which means all nodes) passed in.
+
+> **Note:**
+> 
+> If `-R, --role` is also specified, only the services in the intersection of the specified nodes and roles is queried.
+
+### -R, --role strings
```
Comment (CharLotteiu) [inline tiup/tiup-component-dm-display.md:29]:
```suggestion
### -R, --role
```

**Review by AstroProfundis — APPROVED** [review]

File: tiup/tiup-component-cluster-check.md [hunk 6]
```diff
@@ -0,0 +1,216 @@
...
+
+- Enables the memory size check.
+- Data type: `BOOLEAN`
+- This option is disabled by default with the `false` value. To enable this option, add this option to the command, and either pass the `true` value or do not pass any value.
+
+### --u, --user
+
+- Specifies the user name to connect to the target machine. The specified user needs to have the password-free sudo root privileges on the target machine.
+- Data type: `STRING`
+- If this option is not specified in the command, user who executes the command is used as the default value.
```
Comment (qiancai) [inline tiup/tiup-component-cluster-check.md:179]:
```suggestion
- If this option is not specified in the command, the user who executes the command is used as the default value.
```

File: tiup/tiup-component-dm-display.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+
+If you want to check the operational status of each component in a cluster, it is inefficient to log in to each machine one by one. Therefore, tiup-dm provides the `tiup dm display` command to do this job efficiently.
+
+## Syntax
+
+```shell
+tiup dm display <cluster-name> [flags]
+```
+
+`<cluster-name>` is the name of the cluster to operate on. If you forget the cluster name, check it in [tiup cluster list](/tiup/tiup-component-cluster-list.md).
```
[the remark this answers is not in the document]
Comment (qiancai) [reply tiup/tiup-component-dm-display.md:15]: @lucklove please help check if a similar change is also needed in the Chinese version. Thanks!

File: tiup/tiup-component-dm-upgrade.md (hunk 1 above)
Comment (qiancai) [reply tiup/tiup-component-dm-upgrade.md:15]: @lucklove please help check if a similar change is also needed in the Chinese version. Thanks!

File: tiup/tiup-component-dm-display.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (lucklove) [reply tiup/tiup-component-dm-display.md:15]: It seems that the Chinese version is correct.

File: tiup/tiup-component-dm-upgrade.md (hunk 1 above)
Comment (lucklove) [reply tiup/tiup-component-dm-upgrade.md:15]: ditto

File: tiup/tiup-component-dm-upgrade.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
+--
+title: tiup dm upgrade
+--
```
Comment (CharLotteiu) [inline tiup/tiup-component-dm-upgrade.md:3]:
```suggestion
---
title: tiup dm upgrade
---
```

It needs three `-` to indicate this part as the metadata of the file.

File: tiup/tiup-component-dm-display.md [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+
+If you want to check the operational status of each component in a DM cluster, it is inefficient to log in to each machine one by one. Therefore, tiup-dm provides the `tiup dm display` command to do this job efficiently.
+
+## Syntax
+
+```shell
+tiup dm display <cluster-name> [flags]
+```
+
+`<cluster-name>` is the name of the cluster to operate on. If you forget the cluster name, check it in [tiup cluster list](/tiup/tiup-component-cluster-list.md).
```
Comment (qiancai) [inline tiup/tiup-component-dm-display.md]:
```suggestion
`<cluster-name>` is the name of the cluster to operate on. If you forget the cluster name, you can check it using the `[tiup cluster list](/tiup/tiup-component-cluster-list.md)` command.
```

File: tiup/tiup-component-dm-upgrade.md [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+
+The `tiup dm upgrade` command is used to upgrade a specified cluster to a specific version.
+
+## Syntax 
+
+```shell
+tiup dm upgrade <cluster-name> <version> [flags]
+```
+
+- `<cluster-name>` is the name of the cluster to be operated on. If you forget the cluster name, check it in [tiup DM cluster list](/tiup/tiup-component-dm-list.md).
```
Comment (qiancai) [inline tiup/tiup-component-dm-upgrade.md:15]:
```suggestion
- `<cluster-name>` is the name of the cluster to be operated on. If you forget the cluster name, you can use the `[tiup dm list](/tiup/tiup-component-dm-list.md)` command to check it.
```

File: tiup/tiup-component-dm-display.md [hunk 11]
```diff
@@ -0,0 +1,58 @@
...
+
+If you want to check the operational status of each component in a DM cluster, it is inefficient to log in to each machine one by one. Therefore, tiup-dm provides the `tiup dm display` command to do this job efficiently.
+
+## Syntax
+
+```shell
+tiup dm display <cluster-name> [flags]
+```
+
+`<cluster-name>` is the name of the cluster to be operated. If you forget the cluster name, check it in the [tiup dm cluster list](/tiup/tiup-component-dm-list.md).
```
Comment (qiancai) [inline tiup/tiup-component-dm-display.md:15]:
```suggestion
`<cluster-name>` is the name of the cluster to be operated. If you forget the cluster name, you can use the `[tiup dm list](/tiup/tiup-component-dm-list.md)` command to check it.
```

File: tiup/tiup-component-dm-display.md [hunk 12]
```diff
@@ -0,0 +1,58 @@
...
+
+`<cluster-name>` is the name of the cluster to be operated. If you forget the cluster name, you can use the `[tiup dm list](/tiup/tiup-component-dm-list.md)` command to check it.
+
+## Options
+
+### -N, --node
+
+Specifies the IDs of the nodes to query, splitting by commas for multiple nodes. If you are not sure about the ID of a node, you can skip this option in the command to show the IDs and status of all nodes in the output.
+Data type: `STRING`
+This option is enabled by default with `[]` (which means all nodes) passed in.
```
Comment (CharLotteiu) [inline tiup/tiup-component-dm-display.md:23]:
```suggestion
 - Specifies the IDs of the nodes to query, splitting by commas for multiple nodes. If you are not sure about the ID of a node, you can skip this option in the command to show the IDs and status of all nodes in the output.
- Data type: `STRING`
- This option is enabled by default with `[]` (which means all nodes) passed in.
```

File: tiup/tiup-component-dm-display.md [hunk 13]
```diff
@@ -0,0 +1,58 @@
...
+
+> **Note:**
+> 
+> If `-R, --role` is also specified, only the services in the intersection of the specified nodes and roles is queried.
+
+### -R, --role
+
+Specifies the roles to query, splitting by commas for multiple roles. If you are not sure about the role deployed on a node, you can skip this option in the command to show the roles and status of all nodes in the output.
+Data type: `STRING`
+This option is enabled by default with `[]` (which means all roles) passed in.
```
Comment (CharLotteiu) [inline tiup/tiup-component-dm-display.md:33]:
```suggestion
- Specifies the roles to query, splitting by commas for multiple roles. If you are not sure about the role deployed on a node, you can skip this option in the command to show the roles and status of all nodes in the output.
- Data type: `STRING`
- This option is enabled by default with `[]` (which means all roles) passed in.
```

**Review by CharLotteiu — COMMENTED** [review]
Please change these contents to lists to pass the manual line break CI check.

File: tiup/tiup-component-dm-display.md [hunk 14]
```diff
@@ -0,0 +1,58 @@
...
+tiup dm display <cluster-name> [flags]
+```
+
+`<cluster-name>` is the name of the cluster to be operated. If you forget the cluster name, you can use the `[tiup dm list](/tiup/tiup-component-dm-list.md)` command to check it.
+
+## Options
+
+### -N, --node
+
+ - Specifies the IDs of the nodes to query, splitting by commas for multiple nodes. If you are not sure about the ID of a node, you can skip this option in the command to show the IDs and status of all nodes in the output.
```
Comment (CharLotteiu) [inline tiup/tiup-component-dm-display.md:21]:
```suggestion
- Specifies the IDs of the nodes to query, splitting by commas for multiple nodes. If you are not sure about the ID of a node, you can skip this option in the command to show the IDs and status of all nodes in the output.
```
Sorry for the additional space 😂

**Pull request merged**